ci_framework/roles/polarion/README.md

```diff
@@ -0,0 +1,21 @@
+# polarion
+Role to setup jump tool and upload XML test results to Polarion.
+
+## Parameters
+* `cifmw_polarion_jump_repo_url`: (String) URL of jump repository
+* `cifmw_polarion_storage_fqdn`: :(String) FQDN of a storage server holds the xml test results files.
+* `cifmw_polarion_storage_ssh_user`: :(String) A username that is used for SSH conection.
```

Comment:
instead of ssh/rsync access, i would expect local path (or maybe url later, though that would need recursive fetching maybe?)